pyccl/core.py

```diff
@@ -82,9 +94,12 @@ def __init__(self, Omega_c=None, Omega_b=None, h=None, A_s=None, n_s=None,
                          specified.
             n_s (float): Primordial scalar perturbation spectral index.
             Omega_k (float, optional): Curvature density fraction. Defaults to 0.
-            N_nu_rel (float, optional): Number of massless neutrinos present. Defaults to 3.046
-            N_nu_mass (float, optional): Number of massive neutrinos present. Defaults to 0.
-            m_nu (float, optional): total mass in eV of the massive neutrinos present (current must be equal mass). Defaults to 0.
+            Neff (float, optional): Effective number of neutrino species. Defaults to 3.046
+            m_nu (float or array-like, optional): If float: total mass in eV of 
+							the massive neutrinos present. If array-like, masses of 3 neutrino
+							species (must have length 3).
+			mnu_type (string): treatment for neutrinos. Available: 'sum', 'sum_inverted',
```

Comment:
Weird indent here? We should always use spaces, not tabs.

Reply:
Also, mention what the default settings are.

Reply:
Good point, fixed.